reactive_graph/src/effect/effect.rs

```diff
@@ -196,6 +198,57 @@ impl Effect<LocalStorage> {
         Self { inner }
     }
 
+    /// Creates a new effect, which runs once on the next “tick”, and then runs again when reactive values
+    /// that are read inside it change.
+    ///
+    /// This spawns a task on the local thread using
+    /// [`spawn_local`](any_spawner::Executor::spawn_local). For an effect that can be spawned on
+    /// any thread, use [`new_sync`](Effect::new_sync).
```

Comment:
Docs should probably be updated to
1) Note that this allows you to pass in a runtime, and 
2) Update the `new_sync` x 2 here to `new_sync_with_runtime`